IHP/IDE/SchemaDesigner/Parser.hs

```diff
@@ -194,6 +212,25 @@ sqlType = choice
                     try (symbol' "TIME")
                     pure PTime
 
+                numericPS = do
+                    try (symbol' "NUMERIC(")
+                    values <- between (space) (char ')' >> space) (varExpr `sepBy` (char ',' >> space))
+                    case values of
+                        [VarExpression precision, VarExpression scale] -> pure (PNumericP (tryTextToInt precision) (tryTextToInt scale))
+                        [VarExpression precision] -> pure (PNumericP (tryTextToInt precision) 0)
+                        _ -> pure PNumeric
```

Comment:
This will silently fail here, we should call `fail` here instead:
```suggestion
                        _ -> fail "Failed to parse NUMERIC(..) expression"
```